lib/web/desktop.go

```diff
@@ -504,25 +498,27 @@ func proxyWebsocketConn(ctx context.Context, ws *websocket.Conn, wds net.Conn, l
 
 	tdpConnProxy := tdp.NewConnProxy(&WebsocketIO{Conn: ws}, wds, func(_ *tdp.Conn, msg tdp.Message) (tdp.Message, error) {
 		if ping, ok := msg.(tdp.Ping); ok {
-			pings <- ping
+			select {
+			case pings <- ping:
```

Comment:
What happens if we get a `tdp.Ping` from a host for which `latencySupported` is false? Won't this block forever since the latency reporter isn't running?

Reply:
The ping from server is only ever send in response to ping send by pinger which is not enabled if `latencySupported` is false

Reply:
You're right, I've added error in this case